### PR TITLE
Update manage-billing-profiles.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the billing profiles to reduce ambiguity

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8751

File: microsoft-365/commerce/billing-and-payments/manage-billing-profiles.md [hunk 1]
```diff
@@ -24,11 +24,7 @@ ms.date: 04/02/2021
 
 # Understand billing profiles
 
-A billing profile contains a payment method, Bill-to information, and other invoice settings, such as purchase order number and email invoice preference. You use a billing profile to pay for the products that you buy from Microsoft. A billing profile is automatically created when a user makes a self-service purchase. Each billing profile is invoiced separately.
-
-> [!NOTE]
->
-> Billing profiles are not available to customers who buy products and services from Microsoft.com or on the **Purchase services** page of the Microsoft 365 admin center.
+A billing profile contains a payment method, Bill-to information, and other invoice settings, such as purchase order number and email invoice preference. You use a billing profile to pay for the products that you buy from Microsoft. A billing profile is automatically created when Billing profiles are only available if you bought your subscription directly through Microsoft Sales. They aren't available for customers who bought their subscription through the products web site or on the Purchase services page in the Microsoft 365 admin center. Each billing profile is invoiced separately.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/billing-and-payments/manage-billing-profiles.md]:
```suggestion
A billing profile contains a payment method, Bill-to information, and other invoice settings, such as purchase order number and email invoice preference. You use a billing profile to pay for the products that you buy from Microsoft. Billing profiles are automatically created. A Billing profile is only available if you bought your subscription directly through Microsoft Sales. They aren't available for customers who bought their subscription through the product's website or on the Purchase services page in the Microsoft 365 admin center. Each billing profile is invoiced separately. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Pull request merged**